shared/actions/platform-specific/index.native.js

```diff
@@ -216,7 +218,8 @@ function* loadStartupDetails() {
   const routeStateTask = yield Saga.fork(AsyncStorage.getItem, 'routeState')
   const linkTask = yield Saga.fork(Linking.getInitialURL)
   const initialPush = yield Saga.fork(getStartupDetailsFromInitialPush)
-  const [routeState, link, push] = yield Saga.join(routeStateTask, linkTask, initialPush)
+  const lastSentXLMTask = yield Saga.fork(AsyncStorage.getItem, 'lastSentXLM')
```

Comment:
we're deprecating AsyncStorage (there's a pr open for this now) so lets not use it

Reply:
ok, done